includes/utils.php

```diff
@@ -299,6 +299,41 @@ function distributable_post_types() {
 	return apply_filters( 'distributable_post_types', array_diff( $post_types, [ 'dt_ext_connection', 'dt_subscription' ] ) );
 }
 
+/**
+ * Return post types that should be excluded from the permission list.
+ *
+ * @since  1.7.0
+ * @return array
+ */
+function get_excluded_post_types_from_permission_list() {
+	// Hide the built-in post types except 'post' and 'page'.
+	$hide_from_list = get_post_types(
+		array(
+			'_builtin'     => true,
+			'show_in_rest' => true,
+		)
+	);
+	unset( $hide_from_list['post'], $hide_from_list['page'] );
+
+	// Default is keyed by the post type 'post' => 'post', etc; hence using `array_values`.
+	$hide_from_list = array_values( $hide_from_list );
+
+	// Hide 'dt_subscription' post type.
+	$hide_from_list[] = 'dt_subscription';
+
+	/**
+	 * Filter to update the list of post types that should be hidden from the "Post types permissions" list.
+	 *
+	 * @since 1.7.0
+	 * @hook dt_hide_post_types_from_permission_list
+	 *
+	 * @param {array} The list of hidden post types.
+	 *
+	 * @return {bool} The updated array with the list of post types that should be hidden.
+	 */
+	return apply_filters( 'dt_hide_post_types_from_permission_list', $hide_from_list );
```

Comment:
Do you think we should update this filter name to match with the function name, or vice versa, update the function name to match the filter name?

Reply:
Consistency is nice. I think renaming the the filter `dt_excluded_post_types_from_permission_list` would be good.